docs/environments/connector.md

```diff
@@ -62,7 +49,7 @@ Agent2[path=4, position=5, target=6]
 Agent3[path=7, position=8, target=9]
 ```
 
-For example, on a 6x6 grid, the starting observation is shown below.
+This would result in a grid like this:
```

Comment:
Isn't the previous description more informative?

Reply:
Technically it's not the starting observation which is why I changed it. Have reworded it better

Reply:
Thank you!